.gitmodules

```diff
@@ -9,6 +9,6 @@
 [submodule "third_party/fairseq"]
 	path = third_party/fairseq
 	url = https://github.com/facebookresearch/fairseq.git
-[submodule "Megatron-DeepSpeed"]
```

Comment:
Do we need to keep the Megatron-DeepSpeed example?

Reply:
I remember we have discussed this at scrum meeting. Peng's suggestion is removing Megatron-DeepSpeed.

Reply:
I think Megatron-DeepSpeed is a good example to show how to integrate MS-AMP + deepspeed.

Reply:
Sure. Let's keep both of Megatron-LM and Megatron-DeepSpeed.